src/lucky/action_callbacks.cr

```diff
@@ -24,14 +27,64 @@ module Lucky::ActionCallbacks
     \{% end %}
   end
 
+  # Run a method before an action starts
+  #
+  # Methods will run in the order that each `before` is defined. Also, each
```

Comment:
Is the order something you guarantee by the way? Right now that's just how the implementation works but I wasn't sure if it was intentional or a side-effect.

If it's intentional maybe we should have a test for it.

Reply:
That's a really good point. It is intentional, but it'd be good to have a test to ensure that it stays that way

Reply:
I can only think of hacky ways to test like having callbacks append to the same file and then checking the line order. Got any better ideas? Can we mock things method calls?

Reply:
I explicitly checked order in LuckyRecord here: https://github.com/luckyframework/lucky_record/blob/master/spec/form_callbacks_spec.cr

An instance var with an array of strings that you append to in each callback. Then you can check that the instance var has the correct strings, e.g. `callbacks_that_run.should eq ["first_before", "second_before"]`

Reply:
That one also explicitly checks that it works for symbols or bare words